Source/Atomic/Atomic2D/ParticleEmitter2D.cpp

```diff
@@ -312,8 +316,7 @@ void ParticleEmitter2D::Update(float timeStep)
             emitParticleTime_ -= timeBetweenParticles;
         }
 
-        if (emissionTime_ > 0.0f)
-            emissionTime_ = Max(0.0f, emissionTime_ - timeStep);
+        emissionTime_ = Max(0.0f, emissionTime_ - timeStep);

```

Comment:
I misunderstood that this is a "bug" from upstream, sorry, yeah, leaving this would be fine in this case, otherwise, please guard it, or even better make a PR upstream with its removal, which would solve both problems :)